api/src/catalog/sitemap.utils.ts

```diff
@@ -0,0 +1,66 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import { Readable } from 'node:stream';
+import { SitemapStream, streamToPromise } from 'sitemap';
+import { getSdk, type CatalogBySeasonQuery } from './catalog.queries.js';
+import { STATIC_FILE_DIR, graphqlClient } from '../config.js';
+import winston from '../logging/winston.js';
+
+interface Course {
```

Comment:
Call this `SitemapData` or `SitemapListing` etc. "Course" means something very specific in our terminology